src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerRole = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      {isMarried
+        ? <p className="Person__partner">{`${partnerName} is my ${partnerRole}`}</p>
+        : <p className="Person__partner">I am not married</p>
+      }
```

Comment:
Consider it will be better and more readable, because we do not duplicate the tag with the same class twice, but write the condition inside immediately
```suggestion
        <p className="Person__partner">
          {isMarried ? `${partnerName} is my ${partnerRole}` : 'I am not married'}
        </p>
```

Reply:
I agree. Done!